test/src/loaders/ncbi_downloader/ncbi_downloader_test.py

```diff
@@ -43,16 +43,15 @@ def test_make_work_dir(setup_and_teardown):
     assert path.parents[0].name == source
 
 
-def test_fetch_gtdb_genome_ids(setup_and_teardown):
+def test_fetch_gtdb_genome_ids(setup_and_teardown, mocker):
     tmp_dir, script_file = setup_and_teardown
 
-    release_ver = '207'
-    genome_ids = ncbi_downloader._fetch_gtdb_genome_ids(release_ver, tmp_dir)
-    assert len(genome_ids) == 317542
+    mock_form_url = mocker.patch('src.loaders.ncbi_downloader.ncbi_downloader._form_gtdb_taxonomy_file_url')
```

Comment:
I don't really understand why this mock is necessary...?

Reply:
with this mock, it only downloads the much smaller `ar` file rather than downloading both `bac` and `ar`. I was hoping this could help to speed up the test overall.

Reply:
I see, ok